src/components/ebay-icon/symbols/pause.marko

```diff
@@ -0,0 +1 @@
+<symbol id="icon-pause" viewBox="0 0 16 16"><g fill="currentColor" fill-rule="evenodd"><path d="M0 0h6v16H0zM10 0h6v16h-6z"></path></g></symbol>
```

Comment:
Should these be added to the `ebay-icon/examples`?

Reply:
I'm not sure. They are not in skin yet so maybe not?

Reply:
Ah ok, let's leave them out then.